app/Http/Middleware/Localization.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Closure;
+use Session;
+use Lang;
+
+class Localization
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Closure  $next
+     * @return mixed
+     */
+    public function handle($request, Closure $next)
+    {
+        $language = Session::get('language');
+        if(!$language) {
+            $language == 'vi';
+        }
+        Lang::setLocale($language);
```

Comment:
```suggestion
        Lang::setLocale($language);
```
thay vì check trong handle, sửa trong file config ngôn ngữ mặc định và ngôn ngữ fallback sang "vi".

Reply:
Tôi fix được rồi nhà ông 

```php
$language = Session::get('language');
Lang::setLocale($language);
        
return $next($request);
```